<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="plot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>760</width>
      <height>371</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>261</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabManual">
     <attribute name="title">
      <string>Manual</string>
     </attribute>
     <widget class="QSlider" name="currentSlider">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-50</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="currentLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Current:</string>
      </property>
     </widget>
     <widget class="QLabel" name="currentValue">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currentUnit">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>mA</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabImpulse">
     <attribute name="title">
      <string>Impulse</string>
     </attribute>
     <widget class="QPushButton" name="impulseButton">
      <property name="geometry">
       <rect>
        <x>168</x>
        <y>30</y>
        <width>71</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Inject</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="impulseMagnitude">
      <property name="geometry">
       <rect>
        <x>98</x>
        <y>20</y>
        <width>57</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimum">
       <number>-50</number>
      </property>
      <property name="maximum">
       <number>50</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="impulseDuration">
      <property name="geometry">
       <rect>
        <x>98</x>
        <y>50</y>
        <width>57</width>
        <height>24</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
     <widget class="QLabel" name="impulseMagnitudeLabel">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>23</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Magnitude:</string>
      </property>
     </widget>
     <widget class="QLabel" name="impulseDurationLabel">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>52</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Duration:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>361</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabConductivities">
     <attribute name="title">
      <string>Conductivities</string>
     </attribute>
     <widget class="QLabel" name="gNaValue">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="gNaLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>gNa:</string>
      </property>
     </widget>
     <widget class="QSlider" name="gNaSlider">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>20</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>200</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="gNaUnit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>mS/cm2</string>
      </property>
     </widget>
     <widget class="QLabel" name="gKLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>46</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>gK:</string>
      </property>
     </widget>
     <widget class="QLabel" name="gKValue">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>46</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="gKSlider">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>46</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="gKUnit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>46</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>mS/cm2</string>
      </property>
     </widget>
     <widget class="QSlider" name="gLSlider">
      <property name="geometry">
       <rect>
        <x>83</x>
        <y>70</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="gLUnit">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>mS/cm2</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>60</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>gL:</string>
      </property>
     </widget>
     <widget class="QLabel" name="gLValue">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabProbabilities">
     <attribute name="title">
      <string>Probabilities</string>
     </attribute>
     <widget class="QCheckBox" name="checkN">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> n  gate</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkM">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> m gate</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkH">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>87</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string> h  gate</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>560</y>
      <width>410</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HodgkinHuxley Simulator 1.0.1 - Copyright 2013 by Robin Rump</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="controlBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>402</y>
      <width>101</width>
      <height>140</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QPushButton" name="pause">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>34</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>||</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>64</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>94</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionJson"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionWelcome"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionJson">
   <property name="text">
    <string>Export to Json</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionWelcome">
   <property name="text">
    <string>Show welcoming page</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
